Makefile

```diff
@@ -85,3 +85,11 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+clean-api-reference: ## Clean all directories in API reference directory, preserve _index.md
+	rm -rf content/en/docs/reference/kubernetes-api/*/
+
+api-reference: clean-api-reference ## Build the API reference pages. go needed
+	mkdir -p content/en/docs/reference/kubernetes-api
```

Comment:
Is line 93 correct?

Reply:
I'm not sure it is necessary, as the directory will most probably exist now and in the future (it actually exists and contains an_index.md file)

Reply:
Right, so you could remove line 93 (make the directory).

Reply:
Done, thanks